config/sync/migrate_plus.migration.stanford_localist_importer.yml

```diff
@@ -494,6 +494,46 @@ process:
           entity_type: taxonomy_term
           ignore_case: true
           source: name
+  image_file:
+    -
+      plugin: skip_on_empty
+      method: process
+      source: photo_url
+    -
+      plugin: file_import
+      destination: constants/image_path
+      id_only: true
+      reuse: true
+  media_tag:
+    plugin: entity_generate
+    value_key: name
+    bundle_key: vid
+    bundle: media_tags
+    entity_type: taxonomy_term
+    ignore_case: true
+    source: constants/image_tag
```

Comment:
You need to define this constant above.

Reply:
You could also use the `default_value` plugin. either works.

Reply:
I used the default value plugin.